src/main/scala/net/wiringbits/myphototimeline/FileOrganizerService.scala

```diff
@@ -67,8 +67,20 @@ class FileOrganizerService(metadataService: MetadataService)(implicit logger: Si
 
   @tailrec
   private def getAvailablePath(parent: os.Path, name: String, suffix: Int = 0): os.Path = {
-    val actualName = if (suffix == 0) name else s"${name}_($suffix)"
-    val path = parent / actualName
+    var path = parent
```

Comment:
It would be far simpler if you update the method signature to receive the filename, and the extension, which you can get with these calls:

```
   sourceFile.ext
   sourceFile.baseName
```

Reply:
Thanks for pointing that out. I will make that change.